tests/torch2/function_hook/quantization/test_quantized_graphs.py

```diff
@@ -0,0 +1,95 @@
+    (ModelDesc("embedding_model", EmbeddingModel, [1, 10]), {}),
+    (ModelDesc("rope_model", RoPEModel, [1, 10]), {"model_type": ModelType.TRANSFORMER}),
+    (
+        ModelDesc(
+            "scaled_dot_product_attention_model",
+            ScaledDotProductAttentionModel,
+            {"query": [1, 8, 16], "key": [1, 8, 16], "value": [1, 8, 16]},
+        ),
+        {},
+    ),
+    (ModelDesc("shared_model", SharedLayersModel, [1, 1, 5, 6]), {}),
+    (ModelDesc("alexnet", test_models.AlexNet, [1, 3, 32, 32]), {}),
+    (ModelDesc("lenet", test_models.LeNet, [1, 3, 32, 32]), {}),
+    (ModelDesc("resnet18", test_models.ResNet18, [1, 3, 32, 32]), {}),
+    (ModelDesc("vgg16", partial(test_models.VGG, "VGG16"), [1, 3, 32, 32]), {}),
+    (ModelDesc("inception", test_models.GoogLeNet, [1, 3, 32, 32]), {}),
+    (ModelDesc("densenet121", test_models.DenseNet121, [1, 3, 32, 32]), {}),
+    (
+        ModelDesc(
+            "inception_v3", partial(test_models.Inception3, aux_logits=True, transform_input=True), [2, 3, 299, 299]
+        ),
+        {},
+    ),
+    (ModelDesc("squeezenet1_1", test_models.squeezenet1_1, [1, 3, 32, 32]), {}),
+    (ModelDesc("shufflenetv2", partial(test_models.ShuffleNetV2, net_size=0.5), [1, 3, 32, 32]), {}),
+    # TODO(AlexanderDokuchaev): too long without disabled tracing of ssd_head - no_nncf_trace()
+    # (ModelDesc("ssd_vgg", test_models.ssd_vgg300, [2, 3, 300, 300]), {}),
+    # (ModelDesc("ssd_mobilenet", test_models.ssd_mobilenet, [2, 3, 300, 300]), {}),
+    (ModelDesc("mobilenet_v2", test_models.mobilenet_v2, [1, 3, 32, 32]), {}),
+    (ModelDesc("mobilenet_v3_small", test_models.mobilenet_v3_small, [1, 3, 32, 32]), {}),
+    (ModelDesc("unet", test_models.UNet, [1, 3, 360, 480]), {}),
+]
+
+
+@pytest.mark.parametrize(
+    ("desc", "quantization_parameters"), TEST_MODELS_DESC, ids=[m[0].model_name for m in TEST_MODELS_DESC]
+)
+def test_quantized_graphs(desc: ModelDesc, quantization_parameters: Dict[str, Any], regen_ref_data: bool):
+    model = desc.model_builder().eval()
+
+    if isinstance(desc.input_sample_sizes, dict):
+        example_input = {}
+        for name, size in desc.input_sample_sizes.items():
+            example_input[name] = torch.ones(size)
+    else:
+        example_input = torch.ones(desc.input_sample_sizes)
+
+    quantization_parameters["advanced_parameters"] = nncf.AdvancedQuantizationParameters(disable_bias_correction=True)
+    quantization_parameters["subset_size"] = 1
+
+    q_model = nncf.quantize(model, nncf.Dataset([example_input]), **quantization_parameters)
+
+    if isinstance(example_input, dict):
+        nncf_graph = build_nncf_graph(q_model, **example_input)
+    else:
+        nncf_graph = build_nncf_graph(q_model, example_input)
+    nx_graph = get_reference_graph(nncf_graph)
+    dot_nncf_graph = to_pydot(nx_graph)
+    ref_file = REF_DIR / f"{desc.model_name}.dot"
+    compare_with_reference_file(str(dot_nncf_graph), ref_file, regen_ref_data)
```

Comment:
What is the difference between torch and torch2 references? Is it possible to use present references?

Reply:
Names of nodes